pkgs/applications/networking/cluster/k3s/update-script.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl gnugrep gnused jq yq-go nix-prefetch
+#!nix-shell -i bash -p curl gnugrep gnused jq yq-go nurl
```

Comment:
```suggestion
#!nix-shell -i bash -p curl gnugrep gnused jq nurl yq-go
```
(Keeping alphabetical order)